src/server/cluster_data_test.cc

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023, DragonflyDB authors.  All rights reserved.
```

Comment:
Should this file be moved to the `cluster/` subdir, along with its accompanying header & implementation files?

Reply:
I did try to do this but I believe that currently I cant, adding it under cluster subdir fails
see function(cxx_test name)

Reply:
Gotcha. I see that that's also how we do it for `journal_test.cc` as well, so I guess that's ok.
We could create a `CMakeLists.txt` file inside, but I guess it's just not worth it.